crates/storage/src/connection/transaction.rs

```diff
@@ -780,7 +780,17 @@ pub(crate) mod dto {
     /// Represents execution resources for L2 transaction.
     #[derive(Copy, Clone, Debug, Default, Deserialize, Serialize, PartialEq, Eq)]
     #[serde(deny_unknown_fields)]
-    pub struct ExecutionResources {
+    pub struct ExecutionResourcesV0 {
+        pub builtins: BuiltinCounters,
```

Comment:
This is still not quite OK I think. `BuiltinCounters` also had some fields added, so that needs to be split into V0 and V1 as well.

Reply:
[48e954f](https://github.com/eqlabs/pathfinder/pull/2115/commits/48e954f0d64ced03a91766a8c1a528326376de1a)